Description: add missing <cstdint> includes to fix build with GCC 13
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: https://github.com/mapbox/mapbox-gl-native/pull/16669
Last-Update: 2023-06-18

--- a/src/3rdparty/mapbox-gl-native/include/mbgl/util/geometry.hpp
+++ b/src/3rdparty/mapbox-gl-native/include/mbgl/util/geometry.hpp
@@ -4,6 +4,8 @@
 #include <mapbox/geometry/point_arithmetic.hpp>
 #include <mapbox/geometry/for_each_point.hpp>
 
+#include <cstdint>
+
 namespace mbgl {
 
 enum class FeatureType : uint8_t {
--- a/src/3rdparty/mapbox-gl-native/include/mbgl/util/string.hpp
+++ b/src/3rdparty/mapbox-gl-native/include/mbgl/util/string.hpp
@@ -4,6 +4,7 @@
 #include <string>
 #include <cassert>
 #include <cstdlib>
+#include <cstdint>
 #include <exception>
 
 // Polyfill needed by Qt when building for Android with GCC
--- a/src/3rdparty/mapbox-gl-native/src/mbgl/gl/stencil_mode.hpp
+++ b/src/3rdparty/mapbox-gl-native/src/mbgl/gl/stencil_mode.hpp
@@ -2,6 +2,8 @@
 
 #include <mbgl/util/variant.hpp>
 
+#include <cstdint>
+
 namespace mbgl {
 namespace gl {
 
